packages/by-name/qemu-tdx-static/package.nix

```diff
@@ -24,11 +24,11 @@ in
   hostCpuTargets = [ "x86_64-softmmu" ];
 })).overrideAttrs
   (previousAttrs: rec {
-    version = "8.2.2";
```

Comment:
Can we unpin this?

Reply:
Not really, as the patches are version-specific, and nixpkgs upstream is already at `9.2.0`